### ISSUE TITLE
a11y > add/remove items to cart buttons should have input type

### ISSUE THREAD

**Issue by ryanschaefer41** [vigetlabs/EcoCart#67]
Each add / remove button on the food items should have `input type="button"` to indicate the function of the button to screenreaders.

![image](https://user-images.githubusercontent.com/23317410/89204922-e0ac2980-d584-11ea-8c03-0a98ca651c4e.png)
